src/test/unittests/Microsoft.ServiceFabric.Services.Remoting.Tests/RemoteExceptionInformationTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
Sorry, did that once but had an issue with my branch and missed copying those changes over when I redid it. Done now.